public/jira/derived/derive.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  DerivedWorkTiming,
+  deriveWorkTiming,
+  WorkTimingConfig,
+} from "./work-timing/work-timing.js";
+import {
+  DerivedWorkStatus,
+  getWorkStatus,
+  WorkStatusConfig,
+} from "./work-status/work-status.js";
+import {
+  normalizeIssue,
+  NormalizeIssueConfig,
+} from "../normalized/normalize.js";
+import { JiraIssue, NormalizedIssue } from "../shared/types.js";
```

Comment:
should these imports be from TypeScript files?

edit: same with the other added files